docker-hub/scim.md

```diff
@@ -46,22 +48,70 @@ Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](h
 
```

Comment:
suggestion: Add identation to the image above so it keeps aligned with step 6

Reply:
updated